src/consensus.rs

```diff
@@ -371,10 +371,17 @@ impl Consensus {
 }
 
 fn handle_committed_entries(
-    entries: Vec<Entry>,
+    mut entries: Vec<Entry>,
```

Comment:
Do we want to mutate this argument, or it is just a side effect?
- if we want to mutate, it probably should be documented, or better - moved into another function
- If it is a side-effect, can we avoid it?

Reply:
No, it is not a side-effect. It is just a convenient shorthand for
```rust
fn foo(bar: u32) -> _ {
let mut bar = bar;
// Some other code
}